stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/group/GroupChatInfoLeaveDialogFragment.kt

```diff
@@ -0,0 +1,66 @@
+package io.getstream.chat.ui.sample.feature.chat.info.group
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import io.getstream.chat.ui.sample.R
+import io.getstream.chat.ui.sample.databinding.GroupChatInfoLeaveDialogFragmentBinding
+
+internal class GroupChatInfoLeaveDialogFragment : BottomSheetDialogFragment() {
```

Comment:
I see that we have the same confirmation dialog for deleting conversations, just with a different icon. So we probably should use this dialog for confirmation in general or confirming changes in chats. Let's name it `ConfirmActionDialogFragment` or `ConfirmChatActionDialogFragment` and make it a general purpose dialog.